### DISCUSSION TITLE
Export / Import

### DISCUSSION THREAD

**Discussion by maedball** [flobz/psa_car_controller#409] [Q&A]
Does someone know how to export the data that was stored? So which files do i need to take?
I am currently trying to move from one system to the other and dont want to lose all my currently stored data

**Comment by Jordan87** [comment]
For export i.e. Trips i see this
![image](https://github.com/user-attachments/assets/1049201a-d9f6-4af2-a21a-1170f82010d9)
But how to import this file, esspecialy in HA addon it's a mystery to me :)
